tests/parsers/test.manifestjson.js

```diff
@@ -415,6 +415,17 @@ describe('ManifestJSONParser version', function() {
     assert.include(errors[0].message, '/version');
   });
 
+  it('should collect a notice on toolkit version values', () => {
+    var addonLinter = new Linter({_: ['bar']});
+    var json = validManifestJSON({version: '1.0.0.0pre0'});
+    var manifestJSONParser = new ManifestJSONParser(json,
+                                                    addonLinter.collector);
+    assert.equal(manifestJSONParser.isValid, true);
+    var notices = addonLinter.collector.notices;
+    assert.equal(notices[0].code, 'PROP_VERSION_TOOLKIT_ONLY');
```

Comment:
Sorry to be a pain, but can you use `constants.PROP_VERSION_TOOLKIT_ONLY.code` here instead? It's just a bit more future-proof (and what we try our best to do) in case the text of the code ever changes.